metadata-db-app/int-test/cmr/metadata_db/int_test/concepts/utils/subscription.clj

```diff
@@ -5,24 +5,33 @@
    [cheshire.core :as json]
    [cmr.metadata-db.int-test.concepts.utils.interface :as concepts]))
 
-(def subscription-metadata
-  "polygon=-18,-78,-13,-74,-16,-73,-22,-77,-18,-78")
+(def subscription-json
+  (json/generate-string
+   {"Name" "someSubscription"
```

Comment:
nit: CMR coding standard has keys in keyword, e.g. `:Name` rather than "Name".

Reply:
It's consistent with service and variable.